ignition/aws/worker.yaml.tmpl

```diff
@@ -46,6 +46,48 @@ storage:
               user: kubelet
             name: service-account-context
           current-context: service-account-context
+    - path: /etc/kubernetes/config/proxy-config.yaml
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
+          apiVersion: kubeproxy.config.k8s.io/v1alpha1
+          bindAddress: 0.0.0.0
+          clientConnection:
+            acceptContentTypes: ""
+            burst: 10
+            contentType: application/vnd.kubernetes.protobuf
+            kubeconfig: /etc/kubernetes/config/proxy-kubeconfig.yaml
+            qps: 5
+          clusterCIDR: ""
+          configSyncPeriod: 15m0s
+          conntrack:
```

Comment:
Do we really need all these settings? Can they be defaults?

Reply:
That is how default config file looks

Reply:
But why do we need everything specified here? can we only have settings that we override in config?